types/index.d.ts

```diff
@@ -0,0 +1,42 @@
+declare module "systemd-journald" {
+  // https://man7.org/linux/man-pages/man7/systemd.journal-fields.7.html
+  export type JournalFields = {
+    message_id?: string,
+    code_file?: string,
+    code_line?: string,
+    code_func?: string,
+    errno?: string,
+    invocation_id?: string,
+    user_invocation_id?: string,
+    syslog_facility?: string,
+    syslog_identifier?: string,
+    syslog_pid?: string,
+    syslog_timestamp?: string,
+    syslog_raw?: string,
+    documentation?: string,
+    tid?: string,
+    unit?: string,
+    user_unit?: string
+  }
+
+  export default class systemd_journald {
+    constructor(defaultFields: JournalFields);
```

Comment:
`defaultFields` is optional and can be omitted. Thus, change it like this?

```suggestion
    constructor(defaultFields?: JournalFields);
```

Just guesstimating here :D

Reply:
my reasoning was that you wouldn't instantiate it and would use the static methods if you didn't want to provide defaults.
if you'd rather it be optional here too I'll accept the change